src/pymordemos/heat.py

```diff
@@ -24,7 +24,7 @@
 import scipy.sparse as sps
 import matplotlib.pyplot as plt
 
-from pymor.discretizations.iosys import LTISystem
+from pymor.basic import *
 from pymor.reductors.bt import BTReductor
 from pymor.reductors.h2 import IRKAReductor
```

Comment:
`from pymor.basic import *` imports `BTReductor` and `IRKAReductor`, so the above two lines can also be removed.